versions/3.0.md

```diff
@@ -127,22 +127,22 @@ In order to preserve the ability to round-trip between YAML and JSON formats, YA
 - Tags MUST be limited to those allowed by the [JSON Schema ruleset](http://www.yaml.org/spec/1.2/spec.html#id2803231).
 - Keys used in YAML maps MUST be limited to a scalar string, as defined by the [YAML Failsafe schema ruleset](http://yaml.org/spec/1.2/spec.html#id2802346).
 
-**Note:** While APIs are described by OpenAPI documents in YAML or JSON format, the API request and response bodies and other content are not required to be JSON or YAML.
+**Note:** While APIs may be described by OpenAPI documents in YAML or JSON format, the API request and response bodies and other content are not required to be JSON or YAML.
 
 ### <a name="documentStructure"></a>Document Structure
 
-An OpenAPI description MAY be made up of a single document.
-However, parts of the description MAY be split into separate documents, at the discretion of the user.
+An OpenAPI document MAY be made up of a single document.
+However, parts of the document MAY be split into separate files, at the discretion of the user.
```

Comment:
+1. My use would be that a "document" is a thing which is a continuous sequence of bytes (e.g. as a file in a file system, or available via some URL in a web server). An API definition consists of one or multiple such documents.